build/common/installer/scripts/tomlparser.rb

```diff
@@ -25,6 +25,7 @@
 @adxDatabaseName = "containerinsights" # default for all configurations
 @logEnableMultiline = "false"
 @logEnableKubernetesMetadata = false
+@logKubernetesMetadataiIncludeFields = "podLabels,podAnnotations,poduid,image"
```

Comment:
podUID

Reply:
change all to podUid to sync with othe places: 
![image](https://github.com/microsoft/Docker-Provider/assets/25573844/3995c70f-2d6c-4046-aa7d-37dcee05dc81)